src/codegen/utilities/interpreter_helpers.py

```diff
@@ -158,6 +158,11 @@ def generate_interpreter_function_call_args(function_metadata):
             function_call_args.append("None")
         elif is_event_function(function_metadata) and param.parameter_name == "callback_function":
             function_call_args.append("callback_method_ptr")
+        elif param.type == "CVIAbsoluteTime":
```

Comment:
Add `and param.direction == "in"`. You will need to use "instantiation lines" to handle the out-parameter for WaitForValidTimestamp.

Reply:
Moved this to be further down in `param.direction == "in"` if-else loop